tests/integration/MailServiceCest.php

```diff
@@ -31,4 +33,30 @@ public function sendMail(IntegrationTester $I)
             $service->send($message);
         }
     }
+
+    public function ensureDotStuffingHappensBeforeQuotePrintableEncode(IntegrationTester $I)
+    {
+        $text_body = <<<EOT
+        Test mail 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890.
+        More text.
+        EOT;
+
+        $service = new SMTPMailService(
+            $I->createSocketConnector(), new NoAuthenticator(), "localhost"
+        );
+
+        $message = new Message(
+            new Address("blip@test.org", "Rasmus åh Schultz"),
```

Comment:
Let's not reference people who are not here in code. He has friends here still :-D